_posts/2016-10-10-devfest-talks-announcement.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "Talks announcement!"
+date:   2016-11-1 23:59:59
+isStaticPost: false
+---
+
+Less than a two weeks left to our annual DevFest and work piles up on our side.
```

Comment:
> and work piles up on our side.

That sounds a bit too negative, even if true. 

I suggest "and we're on the last stretch to make it an awesome event yet again."